Tests failing

The test `test_projection` is failing for the current commit and the current release as well (1.3.0).

I'm getting this:
```
        torch.testing.assert_close(v_viewmats, _v_viewmats, rtol=1e-3, atol=1e-3)
        torch.testing.assert_close(v_quats, _v_quats, rtol=2e-1, atol=1e-2)
>       torch.testing.assert_close(v_scales, _v_scales, rtol=1e-1, atol=2e-1)
E       AssertionError: Tensor-likes are not close!
E       
E       Mismatched elements: 1 / 335355 (0.0%)
E       Greatest absolute difference: 0.7041703462600708 at index (20956, 1) (up to 0.2 allowed)
E       Greatest relative difference: 1.711435079574585 at index (20956, 1) (up to 0.1 allowed)

tests/test_basic.py:264: AssertionError
========================================================================================================== warnings summary ==========================================================================================================
tests/test_basic.py::test_quat_scale_to_covar_preci[False]
tests/test_basic.py::test_rasterize_to_pixels[3]
  /home/rg/miniconda3/envs/nerfacc/lib/python3.9/site-packages/torch/utils/cpp_extension.py:1967: UserWarning: TORCH_CUDA_ARCH_LIST is not set, all archs for visible cards are included for compilation. 
  If this is not desired, please set os.environ['TORCH_CUDA_ARCH_LIST'].
    warnings.warn(

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
====================================================================================================== short test summary info =======================================================================================================
FAILED tests/test_basic.py::test_projection[True-True-False] - AssertionError: Tensor-likes are not close!
```

--- a/tests/test_basic.py
+++ b/tests/test_basic.py
@@ -239,11 +239,11 @@ def test_projection(test_data, fused: bool, calc_compensations: bool, ortho: boo
         )
 
     # backward
-    v_means2d = torch.randn_like(means2d) * radii[..., None]
-    v_depths = torch.randn_like(depths) * radii
-    v_conics = torch.randn_like(conics) * radii[..., None]
+    v_means2d = torch.randn_like(means2d) * valid[..., None]
+    v_depths = torch.randn_like(depths) * valid
+    v_conics = torch.randn_like(conics) * valid[..., None]
     if calc_compensations:
-        v_compensations = torch.randn_like(compensations) * radii
+        v_compensations = torch.randn_like(compensations) * valid
     v_viewmats, v_quats, v_scales, v_means = torch.autograd.grad(
         (means2d * v_means2d).sum()
         + (depths * v_depths).sum()

